app/Dockerfile

```diff
@@ -5,13 +5,14 @@ COPY package*.json /app/
 FROM base AS builder
 RUN npm install
 COPY . .
+RUN npm run build
 
 FROM builder AS dev
-CMD [ "npm", "start" ] 
+CMD [ "npm", "start" ]
 
-FROM builder AS test
-ENV CI true
-CMD [ "npm", "test" ]
-
-FROM builder AS lint
-CMD [ "npm", "run", "lint", "src/" ]
\ No newline at end of file
+FROM nginx:alpine AS prod
```

Comment:
Could you modify the readme to indicate how to run this image?

Reply:
Sure